src/encoder.rs

```diff
@@ -415,16 +410,15 @@ impl PartialInfo {
     /// Converts this partial info to an owned Info struct,
     /// setting missing values to their defaults
     fn to_info(&self) -> Info<'static> {
-        Info {
-            width: self.width,
-            height: self.height,
-            bit_depth: self.bit_depth,
-            color_type: self.color_type,
-            frame_control: self.frame_control,
-            animation_control: self.animation_control,
-            compression: self.compression,
-            ..Default::default()
-        }
```

Comment:
Was this change actually necessary, or is it just for stylistic reasons?

Reply:
Necessary, as the `..Default::default()` syntax is incompatible with private members.